.github/workflows/build.yaml

```diff
@@ -17,16 +17,12 @@ jobs:
     - run: |
         git fetch --prune --unshallow
         mkdir build
-    - name: Release space from worker
-      run: |
-        sudo rm -rf /usr/local/lib/android # will release about 10 GB if you don't need Android
-        sudo rm -rf /usr/share/dotnet # will release about 20GB if you don't need .NET
     - name: Download meta 🔧
       uses: luet-lab/luet-github-action@master
       with:
         FINAL_REPO: quay.io/kairos/packages
         REPOSITORY_TYPE: docker
-        downloadAllMeta: true
+        downloadAllMeta: false
```

Comment:
that goes back to: https://github.com/Luet-lab/luet-github-action/blob/d8cbcf3057535de7ea33896b8ed44fc7b122d6b0/main.go#L371